Generate both global styles and classes by default (#71)
tailwindlabs/tailwindcss-forms: README.md

More customization examples and best practices coming soon.

### Using classes to style

> In addition to the base styles to automatically style each form element we also generate classes that you can use explicitly. This allows you to style form elements by default and have the ability to style non-form things to more closely match those form elements.

Reviewer: Possible rewording...
```suggestion
In addition to the global styles, we also generate a set of corresponding classes which can be used to explicitly apply the form styles to an element. This can be useful in situations where you need to make a non-form element, such as a `<div>`, look like a form element.
```

Reviewer 2: Oh that's much, much better. 👍

| `select[multiple]`        | `form-multiselect` |
| `[type='checkbox']`       | `form-checkbox`    |
| `[type='radio']`          | `form-radio`       |

> ### Using only base styles or only classes

Reviewer: I feel like the word "global" here is better 🤷‍♂️ 
```suggestion
### Using only global styles or only classes
```

Although we recommend thinking of this plugin as a "form reset" rather than a collection of form component styles, in some cases our default approach may be too heavy-handed, especially when integrating this plugin into existing projects.

> For situations where the default strategy of generating both the base styles *and* classes doesn't work well with your project, you can use `strategy` option to control whether or not form styling is _opt-in_ or applied globally:

Reviewer: ```suggestion
If generating both the global (base) styles and classes doesn't work well with your project, you can use the `strategy` option to limit the plugin to just one of these approaches.
```

```js
// tailwind.config.js
plugins: [

>   require("@tailwindcss/forms")({
>     strategy: 'base', // or strategy: 'class',
>   }),

Reviewer: Is this maybe more clear? 🤔 
```suggestion
  require("@tailwindcss/forms")({
    strategy: 'base', // only generate global styles
    strategy: 'class', // only generate classes
  }),
```

Reviewer 2: I went back and forth on this one before pushing the code :joy:

],
```

> When using the `base` strategy, form elements are styled globally by default.
> 
> When using the `class` strategy, form elements do not receive any reset styles by default, and reset styles are added per element using a new set of `form-*` classes generated by the plugin:

Reviewer: ```suggestion
When using the `base` strategy, form elements are styled globally, and no `form-{name}` classes are generated.

When using the `class` strategy, form elements are not styled globally, and instead must be styled using the generated `form-{name}` classes.
```
